### PR TITLE
[FIX] l10n_latam_invoice_document: No gaps in the sequence for LATAM.

### PR DESCRIPTION
**Pull request by mem-adhoc**
Task: 32588

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by RicGR98** [comment]
Hello,

Could you adapt your PR to fit https://github.com/odoo/odoo/pull/145173/files ?

Thanks :)

File: addons/account/models/account_journal_dashboard.py
Comment (zaoral) [inline addons/account/models/account_journal_dashboard.py:124]: move this to l10n_latam_invoice_document model.

**Review by zaoral — COMMENTED** [review]
+1

File: addons/l10n_latam_invoice_document/models/account_journal.py
Comment (RicGR98) [inline addons/l10n_latam_invoice_document/models/account_journal.py:41]: Empty line

**Review by RicGR98 — COMMENTED** [review]
Thanks for the changes, just one minor thing left.

I see that you're 136+ commits behind, could you also please rebase to fix Runbot's errors ? Thanks again :)

**Comment by mem-adhoc** [comment]
Hello @RicGR98, I already did the rebase of the PR, could you mix it? Thanks, greetings!

**Review by RicGR98 — COMMENTED** [review]
Could you rename your commit to:
- have [FIX] instead of [ADD] (which is for new modules)
- remove the mention of Argentina since this is for LATAM apparently ?

**Comment by mem-adhoc** [comment]
Hello @RicGR98, how are you? I already made the changes you asked me for!

**Comment by RicGR98** [comment]
The commit has to be renamed too

**Comment by mem-adhoc** [comment]
@RicGR98 Sorry, you were right. Changes made!

**Review by RicGR98 — COMMENTED** [review]
Thanks for the changes, I'll forward the PR to the final reviewer :)

**Review by RicGR98 — APPROVED** [review]

**Review by oco-odoo — CHANGES_REQUESTED** [review]
The commit message needs more explanations. I understand what you are doing, but you should rather explain the why ; that's what I need to know ^^ If this is related to some legal need, you should also link the documentation about it. We need all elements.  

Also, the task id you give in the commit message does not correspond to anything in our pipe, so please remove it, as it's more confusing than anything else. We usually put it on PR made from internal tasks just for ease of reference in our workflow.

**Comment by zaoral** [comment]
Hi @oco-odoo 
Thank you for your help.

In this case, this PR is a copy of one already made by @jco-odoo in branch 17.0  https://github.com/odoo/odoo/pull/145173. The one mentioned by @RicGR98. It is a backport of the same fix. Maybe changing the title of the PR to BACKPORT will be better?  

I have just copied the commit message from the one in the 17.0 version to this one, so the commit has more detailed information. Please could you review and let us know if this is ok? or if you think a more detailed functional explanation is needed also let us know. 

This PR was created because we previously discussed it with the LATAM team, in this case with @vbe-odoo who are aware of the problem.

About the task number, this is the reason we did it: we work together on PRs like this one and we take into account errors/tickets/tasks from different sources, for that reason we manage different numbers: Adhoc task, Latam task (Odoo's Latam instance for the official maintainers), Odoo task (internal task) or OPW number. If there is a task related to the PR on any of the mentioned Odoo instances, we add all the related numbers to the PR. That is why you see that number there. If it is ok then that we leave the number on the commit? or maybe only leave it in the PR description?

New commit preview 
```
[[FIX] l10n_latam_invoice_document: the gaps message does not make sen
…se for LATAM journals

As the vendor bill names are those from the vendors
and the customer invoices are per document type,
it does not make a lot of sense to care about the gaps
in the traditional sense for LATAM journals.

Task's references: LATAM 1153, ADHOC 32588
```

Let me know what you think
Best Regards!

**Comment by oco-odoo** [comment]
Hi @zaoral :) 
Much better already ; since it's a backport, could I just ask you to add it in the commit message, with a link to the original commit  ? (like, at the end of it, it's fine) I'll merge it right after.

Of the references at the end, it's fine as you did it in the current version of the commit.

**Comment by vbe-odoo** [comment]
Just in case, I tested and LGTM.
Thanks!
cc' @oco-odoo @zaoral

**Comment by mem-adhoc** [comment]
Hi @oco-odoo! I already put the original PR in the commit message! Thanks, greetings.

**Review by oco-odoo — APPROVED** [review]
@robodoo r+

Thanks :)

**Pull request closed without merging**
